site/src/hooks/useTab.ts

```diff
@@ -1,13 +1,13 @@
 import { useSearchParams } from "react-router-dom"
 
 export interface UseTabResult {
-  value: string | null
+  value: string
   set: (value: string) => void
 }
 
-export const useTab = (tabKey: string): UseTabResult => {
+export const useTab = (tabKey: string, defaultValue: string): UseTabResult => {
```

Comment:
Should we assign a default value here in case the consumer doesn't pass one in?

Reply:
We don't know what value it can be so I think we can left it required for now.